test/libsolidity/semanticTests/externalSource/non_normalized_paths.sol

```diff
@@ -0,0 +1,28 @@
+==== ExternalSource: _non_normalized_paths/a.sol ====
+==== ExternalSource: _non_normalized_paths//b.sol ====
+==== ExternalSource: C/////c.sol=_non_normalized_paths/c.sol ====
+==== ExternalSource: C/../////D/d.sol=_non_normalized_paths///d.sol ====
+import "_non_normalized_paths/a.sol";
+import "_non_normalized_paths/b.sol";
+import "C/c.sol";
+import "D/d.sol";
```

Comment:
The point of the test I suggested was that if you have `ExternalSource: a///b.sol` and `ExternalSource: a/b.sol` then `import "a///b.sol"` should import the one with `///`. That's how it currently works in `CompilerStack` - import paths are not normalized in anyway and I think we want this to behave in tests like that too.